addons/sale_project/models/sale_order_line.py

```diff
@@ -78,13 +78,16 @@ def _compute_qty_delivered_method(self):
         milestones_lines.qty_delivered_method = 'milestones'
         super(SaleOrderLine, self - milestones_lines)._compute_qty_delivered_method()
 
-    @api.depends('qty_delivered_method', 'product_uom_qty', 'reached_milestones_ids.quantity_percentage')
```

Comment:
`qty_delivered_method` removed because it's already in the depends of the original compute method (see `/sale/models/sale_order_line.py` `_compute_qty_delivered`)